extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -2032,7 +2049,7 @@ static public function disableWiki( $wikiId, $flags, $reason = '' ) {
 	 */
 	static private function loadVariableFromDB( $cv_id, $cv_name, $city_id, $master = false ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)